pkg/noobaaaccount/reconciler.go

```diff
@@ -305,6 +305,16 @@ func (r *Reconciler) ReconcileDeletion() error {
 			return fmt.Errorf("failed to delete account %q. got error: %v", r.NooBaaAccount.Name, err)
 		}
 	} else {
+		if _, exists := r.NooBaaAccount.GetAnnotations()["remote-operator"]; exists {
```

Comment:
check if `Annotations` map exist, since it's optional

Reply:
see `GetAnnotationValue` that Amit has added here. can be useful

Reply:
Updated